File: Team5/Akanksha_Singh/Week-1/MissingNumber.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int n1=0,n2=0;
       for(int i=0;i<nums.size();i++) 
       {
          n1+=nums[i];
          n2+=i+1;
       }
        return n2-n1;
    }
};

File: Team1/Riya Tanwar/MissingNumber.java
class Solution {
    public int missingNumber(int[] nums) {
        int arrSum = 0;
        int fullSum = 0;
        for (int i = 0; i < nums.length; i++) {
            arrSum += nums[i];
            fullSum += i + 1;
        }
        return fullSum - arrSum;
    }
}